amplicon-pooling.ipynb

```diff
@@ -523,9 +523,20 @@
    "execution_count": 6,
    "metadata": {},
    "outputs": [],
+   "source": [
+    "seq_type = 'ITS'\n",
+    "#['16S', '18S', 'ITS']\n",
```

Comment:
I think moving this to the top, and indicating to the readers that this variable is "global" by typing it in all upper-case would be a good idea. Also, this is a good opportunity to use the Jupyter notebook's rich text formatting to describe what this is and how it will affect the outputs.